src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,36 +21,49 @@ public class DeleteCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameters: INDEX [OPTIONAL INDEXES] (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
```

Comment:
Similarly, maybe it would be better to have another example of multiple indexes like "delete 1 2" (written in the non-magic-literal format ofc)

Reply:
Will update in a later commit 👍